### DISCUSSION TITLE
Nice project!

### DISCUSSION THREAD

**Discussion by pawamoy** [jcayers20/mkdocs-autoapi#58] [General]
Hey, nice project!

I have mentioned it here: https://github.com/mkdocstrings/mkdocstrings/issues/389. And I'm planning on listing it in mkdocstrings' docs too :slightly_smiling_face: Would that be OK with you?

**Comment by jcayers20** [comment]
I appreciate the kind words! This project wouldn't be possible without the amazing work that you and others have done on mkdocstrings, so thank you!

You're more than welcome to list in mkdocstrings'  docs - thanks for checking with me first :smile:

**Reply by pawamoy** [reply]
OK great! I'll let you know once I've listed it :relaxed: Thanks for your work on mkdocs-autoapi!
